packages/cli/src/commands/setup/auth/auth.js

```diff
@@ -68,13 +68,17 @@ const addWebImports = (content, imports) => {
 // returns the content of App.{js,tsx} with init lines added (if there are any)
 const addWebInit = (content, init) => {
   if (init) {
-    return content.replace(
-      'const App = () => (',
-      `${init}\n\nconst App = () => (`
-    )
-  } else {
-    return content
+    const regex = /(const App = \(.*\) => [\(\{])/
```

Comment:
@Tobbe thoughts on this one? I left a comment in PR description

Reply:
Sorry, didn't realize you were waiting for an answer. 

```suggestion
    const regex = /const App = \(.*\) => [({]/
```

You don't need the capture group here. And you don't need to escape `(` and `{` inside the set.

That was the easy part 🙂 

The bigger question here is how flexible we need to be. You already mentioned not supporting
```
function App() {
```
There is also this one:
```
const App = ({
  reallyLongPropName1,
  reallyLongPropName2,
  reallyLongPropName3,
  reallyLongPropName4,
}) => {}
```
And for those who are using TS and like to use `React.FC`:
```
const App: React.FC<Props> = (props) => {
```
Or if you want to be explicit about return value
```
const App = ({ prop1, prop1 }: Props): ReactElement => {
```
Or if you're weird about spacing
```
const App=(props)=>{
```

All that said, I don't think we need to worry about any of those right now. I say let's just handle those when/if they come up.

So just clean up the regex a bit and I'm happy 🙂

Reply:
🤯

Reply:
pushed